src/main/java/com/minecolonies/coremod/entity/pathfinding/MinecoloniesAdvancedPathNavigate.java

```diff
@@ -180,6 +180,10 @@ public PathResult setPathJob(
             desiredPos = dest;
             desiredPosTimeout = 50 * 20;
         }
+        else if (overrideDest)
```

Comment:
this would then simply become a if (safeDest) then set the pos, atm it is a weird logic to set the desiredPos to null, since this only triggers if the dest is null